pyfolio/tears.py

```diff
@@ -1347,7 +1358,7 @@ def create_perf_attrib_tear_sheet(returns,
             ax=plt.subplot(gs[current_section])
         )
 
-    gs.tight_layout(fig)
+    # gs.tight_layout(fig)
```

Comment:
Why?

Reply:
The tight_layout will make the legend majority part out of scope while savefig into pdf or png when I ran the tear function.